main.go

```diff
@@ -232,12 +239,60 @@ func main1() int {
 	return 0
 }
 
+func checkGoVersion() bool {
+	out, err := exec.Command("go", "version").CombinedOutput()
+	if err != nil {
+		fmt.Printf(`Can't get go version: %v
+
+Possible reasons:
+1. Golang not installed
+2. Environment variables $PATH or $GOROOT set incorrectly
+3. Unsupported golang version installed (supported %s)
```

Comment:
This is not possible if some version of go is installed the command will not error.